Jenkinsfile-rf-pipeline

```diff
@@ -0,0 +1,308 @@
+                    if (fileExists("${env.WORKSPACE}/NeedNewImage.txt") || ("${env.GENERATE_IMAGE}" == "true")) {
+                        // Set stage specific vars
+                        pipelineUtils.setStageEnvVars(currentStage)
+
+                        // Set our message topic, properties, and content
+                        (topic, messageProperties, messageContent) = pipelineUtils.setMessageFields("image.running")
+
+                        // Send message org.centos.prod.ci.pipeline.image.running on fedmsg
+                        sendMessage(messageProperties, messageContent)
+
+                        // Provision resources
+                        pipelineUtils.provisionResources(currentStage)
+
+                        // Stage resources - ostree image compose
+                        pipelineUtils.setupStage(current_stage, 'fedora-atomic-key')
+
+                        // Rsync Data
+                        pipelineUtils.rsyncData(currentStage)
+
+                        ostree_props = "${env.ORIGIN_WORKSPACE}/logs/ostree.props"
+                        ostree_props_groovy = "${env.ORIGIN_WORKSPACE}/ostree.props.groovy"
+                        pipelineUtils.convertProps(ostree_props, ostree_props_groovy)
+                        load(ostree_props_groovy)
+
+                        // Teardown resources
+                        pipelineUtils.teardownResources(currentStage)
+
+                        // Set our message topic, properties, and content
+                        (topic, messageProperties, messageContent) = pipelineUtils.setMessageFields("image.complete")
+
+                        // Send message org.centos.prod.ci.pipeline.image.complete on fedmsg
+                        sendMessage(messageProperties, messageContent)
+
+                    } else {
+                        echo "Not Generating a New Image"
+                    }
+                }
+
+                currentStage = "ci-pipeline-ostree-image-boot-sanity"
+                stage(currentStage) {
+                    if (fileExists("${env.WORKSPACE}/NeedNewImage.txt") || ("${env.GENERATE_IMAGE}" == "true")) {
+                        pipelineUtils.setStageEnvVars(currentStage)
+
+                        // Set our message topic, properties, and content
+                        (topic, messageProperties, messageContent) = pipelineUtils.setMessageFields("smoke.running")
+
+                        // Send message org.centos.prod.ci.pipeline.smoke.running on fedmsg
+                        sendMessage(messageProperties, messageContent)
+
+                        // Provision resources
+                        pipelineUtils.provisionResources(currentStage)
+
+                        // Stage resources - ostree image boot sanity
+                        pipelineUtils.setupStage(current_stage, 'fedora-atomic-key')
+
+                        // Rsync Data
+                        pipelineUtils.rsyncData(currentStage)
```

Comment:
You changed the name of the method so this needs to be updated

Reply:
rsyncData() didn't change name. The method rsyncResults was renamed to runTaskAndReturnLogs(). 
This wouldn't need to be updated.

Reply:
Then why do we need runTaskAndReturnLogs() for?

Reply:
kk